.buildkite/scripts/health-report-tests/scenario_executor.py

```diff
@@ -0,0 +1,65 @@
+"""
+A class to execute the given scenario for Logstash Health Report integration test
+"""
+import time
+from logstash_health_report import LogstashHealthReport
+
+
+class ScenarioExecutor:
+    logstash_health_report_api = LogstashHealthReport()
+
+    def __init__(self):
+        pass
+
+    def __has_intersection(self, expects, results):
+        # we expect expects to be existing in results
+        for expect in expects:
+            for result in results:
```

Comment:
Recently `help_url` was added and we need to reflect it here, at least to check the page availibility.
```suggestion
            for result in results:
                if result.get('help_url') and "health-report-pipeline-status.html#" not in result.get('help_url'):
                    return False
```

Reply:
Locally tested, pushing with suggestions.